packages/sdk-client/src/index.ts

```diff
@@ -1,5 +1,5 @@
 export { default as ClientBuilder } from './client-builder/ClientBuilder'
-export { default as createClient } from './sdk-client/client'
+export { default as createClient, process as Process } from './sdk-client/client'
```

Comment:
Any reason why it's been exported from the package with a capital "Process". I'm guessing the `process` function can be chained

Reply:
Had to make it `Process` with a capital `P` so it doesn't get shadowed by the node `process` object.

Sorry I don't understand what you mean by the `process` function can be chained, please elaborate more on this.

Thanks.